components/web_discovery/browser/document_extractor/Cargo.lock

```diff
@@ -0,0 +1,767 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Cargo.lock should not be added, we are only using Cargo.toml for gnrt. This creates confusion about versions, etc...